client/src/helpers/formatNumbers.js

```diff
@@ -0,0 +1,16 @@
+export const internationalFormat = (value) => {
+  return new Intl.NumberFormat().format(Math.round(value * 10) / 10)
+}
+
+export const abbreviateNumbers = (value) => {
```

Comment:
We have a function for this on refine-bio that might be more complete https://github.com/AlexsLemonade/refinebio-frontend/blob/dev/src/common/helpers.js#L347-L365